eks/cluster-loader/clusterloader2/cluster-loader.gotmpl

```diff
@@ -46,17 +46,34 @@ metadata:
 spec:
   template:
     spec:
+      initContainers:
+        - name: clusterloader2
+          image: {{ .Image }}
+          command: ["/clusterloader"]
+          args: {{ .TestArgs }}
+          volumeMounts:
+            - name: config
+              mountPath: /etc/config
+            - name: reports
+              mountPath: /var/reports
+              readOnly: false
       containers:
-      - name: clusterloader2
-        image: {{ .Image }}
-        command: ["/clusterloader"]
-        args: {{ .TestArgs }}
-        volumeMounts:
-        - name: config
-          mountPath: /etc/config
+        - name: uploadresults
+          image: amazon/aws-cli:latest
+          command: 
+            - /bin/sh
+            - -c
+            - >
+              aws s3 cp /var/reports/cluster-loader {{ .S3Uri }} --recursive        
+          volumeMounts: 
+            - name: reports
+              mountPath: /var/reports
+              readOnly: true       
```

Comment:
What's the default on this field? I'd omit readOnly if the value you set is the same as the default.

Reply:
Fair enough.